ra/ra.go

```diff
@@ -275,7 +275,11 @@ func NewRegistrationAuthorityImpl(
 }
 
 func (ra *RegistrationAuthorityImpl) SetRateLimitPoliciesFile(filename string) error {
```

Comment:
nit: Same very minor and optional comment here -- this function now loads the file, not just sets the name of the file to be loaded.